docs/references/astro/locals.mdx

```diff
@@ -48,14 +47,11 @@ if (Astro.request.method === "POST") {
   <button type="submit">Add to Cart</button>
 </form>
 ```
+</CodeBlockTabs>
 
 ## Accessing the current user
 
-Current user data is important for data enrichment. You can use the `currentUser()` local to fetch the current user's data in your pages.
-
-Under the hood, this local:
-- uses the `GET /v1/users/me` endpoint.
-- counts towards the [Backend API Request Rate Limit](/docs/backend-requests/resources/rate-limits#rate-limits).
```

Comment:
We should be keeping some warning about this local counting towards rate limits. We do that, I think, everywhere else that the Next version of `currentUser()` is documented.

Reply:
updated!